src/components/button/_index.scss

```diff
@@ -0,0 +1,64 @@
+@use "../../scss";
+
+.c-button {
+	-webkit-appearance: none;
+	align-items: center;
+	display: inline-flex;
+
+	@include scss.component-properties("button");
+
+	&:hover {
+		cursor: pointer;
+
+		@include scss.component-properties("button-hover");
+	}
+
+	&:active {
+		@include scss.component-properties("button-active");
+	}
+
+	&:focus {
+		@include scss.component-properties("button-focus");
+	}
+
+	&[aria-disabled] {
```

Comment:
It's really good that we're using `aria-disabled` to ensure screen readers can see the button. Should we also have a reduced opacity in this selector like the effect of instead using the `disabled` attribute?  If not, are we relying on a token for this?

```
&[aria-disabled] {
  cursor: not-allowed;
  opacity: 0.5
}
```

Here's a reference I found for this:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-disabled

Reply:
We should avoid putting too much style in to the base component - I'd like to rely on the token system here for appearance